Samples/iOS-ObjectiveC/iOS-ObjectiveC/ViewController.m

```diff
@@ -78,6 +78,28 @@ - (IBAction)captureUserFeedback:(id)sender
     [SentrySDK captureUserFeedback:userFeedback];
 }
 
+- (IBAction)captureNewUserFeedback:(id)sender
```

Comment:
`l`: Using V2 instead of new for naming is usually better. What if we have user feedback V3? Are we going to call it newnew then?

Reply:
oh yeah, we talked about this before with the profiling stuff 🤣 I'll change it to V2